Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp

```diff
@@ -1018,6 +1018,11 @@ IPCCommandResult CWII_IPC_HLE_Device_net_ip_top::IOCtl(u32 _CommandAddress)
 					unhandled_events &= ~map[1];
 				}
 
+				// IOS does not care about POLLHUP being passed in;
```

Comment:
Define "does (not) care"?

This comment doesn't seem to be helpful currently, since It looks to be assuming that the reader already knows what the line that you're explaining is trying to do.